mmv1/products/workstations/WorkstationConfig.yaml

```diff
@@ -284,6 +311,8 @@ properties:
         name: "workingDir"
         description: |
           If set, overrides the default DIR specified by the image.
+        # Allow unsetting to revert to container default.
+        send_empty_value: true
```

Comment:
Can we add or modify a test case to test this behavior?

Reply:
Added another test case for this.